.github/workflows/test.yml

```diff
@@ -19,6 +19,8 @@ jobs:
       NO_ET: 1
       DANDI_ALLOW_LOCALHOST_URLS: 1
       DANDI_TESTS_PERSIST_DOCKER_COMPOSE: "1"
+      DJANGO_DANDI_WEB_APP_URL: http://localhost:8085
+      DJANGO_DANDI_API_URL: http://localhost:8000
```

Comment:
What exactly is the point of setting the environment variables here and in `tox.ini`?  Aren't these only used by the Django instance?  Why do they need to be set on the client side, especially considering that (currently) nothing in the client uses them?

Reply:
Yeah you're right, I removed those.

(sorry forgot to submit this comment last week, doing so now just to close this thread)